blogs/tekton-oadp-nonadmin/install_templates/templates/03-rbac-pipeline-role.yaml

```diff
@@ -17,9 +23,21 @@ objects:
     name: oadp-nonadmin-role
   rules:
   - apiGroups:
+    # By default restore resources will NOT be provided by this template.
+    # It is recommended that restores are executed by OpenShift Admin's in production environments.
+    # For demonstration purposes only you may uncomment the restore resources.
     - velero.io
     resources:
-    - '*'
+    - 'backuprepositories'
+    - 'backups'
+    - 'deletebackuprequests'
+    # - 'downloadrequests'
+    - 'podvolumebackups'
+    - 'podvolumerestores'
```

Comment:
Feel like this should be same as restores

Reply:
aye of course.. too fast

Reply:
Hmm. But again, there is a difference. The tekton/non-admin application will need to create backups and restores, but won't need to create PVBs/PVRs, so the permission level for this application could be lower for those.